test/library/packages/UnitTest/test_dependency.chpl

```diff
@@ -0,0 +1,14 @@
+use UnitTest;
+proc A(test: Test) throws {
+  test.assertTrue(true);
+}
+
+proc B(test: Test) throws {
+  test.dependsOn(A);
+}
+
+proc C(test: Test) throws {
+  test.dependsOn(A,B);
```

Comment:
A test that shows the other order would also be good

Reply:
Sure, added it.